sup3r/preprocessing/feature_handling.py

```diff
@@ -896,6 +896,84 @@ def compute(cls, data, height):
         return data['tas'] - 273.15
 
 
+class TasMin(DerivedFeature):
```

Comment:
When I see all these tas features using the same celsius conversion I wonder if there is way to combine them. But maybe its better for readability to keep them seperate? Whats your philosophy on this?

Reply:
No philosophy just didn't think about it, good call though, combined with the help of a class attribute